lib/Extension/config/report/renderers.php

```diff
@@ -11,14 +11,6 @@
  */
 
 return [
-    'html' => [
-        'renderer' => 'xslt',
-        'template' => __DIR__ . '/../../../Report/Renderer/templates/html.xsl',
-    ],
-    'markdown' => [
```

Comment:
removed these temporarily - probably need to write a small PHP templating engine to reintroduce them